### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by Kimkyungmin123** [twentiethcenturygangsta/cassette-client#50]
## Description
`info  - Linting and checking validity of types ...error - ESLint: Failed to load plugin 'jsx-a11y' declared in '.eslintrc.json » eslint-config-next/core-web-vitals » C:\Users\Home_Work\Desktop\cassette-client\.yarn\__virtual__\eslint-config-next-virtual-8c8fb04640\0\cache\eslint-config-next-npm-13.0.6-dbb26ea96a-3d12c5f0cc.zip\node_modules\eslint-config-next\index.js': Required unplugged package missing from disk. This may happen when switching branches without running installs (unplugged packages must be fully materialized on disk to work).  Missing package: core-js-pure@npm:3.26.1 Expected package location: C:\Users\Home_Work\Desktop\cassette-client\.yarn\unplugged\core-js-pure-npm-3.26.1-d78c7431c6\node_modules\core-js-pure\  Require stack: - C:\Users\Home_Work\Desktop\cassette-client\.yarn\cache\@babel-runtime-corejs3-npm-7.20.6-03b445effa-d533d43221.zip\node_modules\@babel\runtime-corejs3\core-js-stable\object\define-property.js - C:\Users\Home_Work\Desktop\cassette-client\.yarn\cache\aria-query-npm-4.2.2-e0c4f1a309-38401a9a40.zip\node_modules\aria-query\lib\index.js - C:\Users\Home_Work\Desktop\cassette-client\.yarn\__virtual__\eslint-plugin-jsx-a11y-virtual-ce9e6d6cbe\0\cache\eslint-plugin-jsx-a11y-npm-6.6.1-02552a3ea2-baae7377f0.zip\node_modules\eslint-plugin-jsx-a11y\lib\rules\aria-activedescendant-has-tabindex.js - C:\Users\Home_Work\Desktop\cassette-client\.yarn\__virtual__\eslint-plugin-jsx-a11y-virtual-ce9e6d6cbe\0\cache\eslint-plugin-jsx-a11y-npm-6.6.1-02552a3ea2-baae7377f0.zip\node_modules\eslint-plugin-jsx-a11y\lib\index.js - C:\Users\Home_Work\Desktop\cassette-client\.yarn\cache\@eslint-eslintrc-npm-1.3.3-9e3a462140-f03e9d6727.zip\node_modules\@eslint\eslintrc\dist\eslintrc.cjs     
info  - Linting and checking validity of types  
warn  - Attempted to load @next/swc-win32-x64-gnu, but it was not installed
warn  - Attempted to load @next/swc-win32-x64-msvc, but it was not installed
error - Failed to load SWC binary for win32/x64, see more info here: https://nextjs.org/docs/messages/failed-loading-swc`
